lambda/executor.js

```diff
@@ -57,6 +57,9 @@ module.exports.handler = async(event, context) => {
         sleepBetweenRunsMs: sleepBetweenRunsMs,
     };
 
+    await utils.waitForAliasActive(lambdaARN, lambdaAlias);
```

Comment:
I'm wondering if we can run this only in case of SnapStart?

We're already invoking the getFunctionConfiguration API above with:
```
const architecture = await utils.getLambdaArchitecture(lambdaARN);
```

We could update that method to fetch both the Architecture and the SnapStart config.

Reply:
Makes sense; will try that approach.

Reply:
ended up fetching the state instead to make it more generic in case some other feature influences time-to-Active